src/static/hint-dark.css

```diff
@@ -0,0 +1,18 @@
+span.dark.TridactylHint {
```

Comment:
Here you should consider applying the dark theme to div.TridactylHintHost and then inheriting from that.